pkg/extractor/extractor.go

```diff
@@ -69,33 +71,49 @@ type FileConfigurations struct {
 
 func ParseYaml(content string) (*[]Configuration, error) {
 	configurations, err := extractYamlConfigurations(content)
-	if err != nil {
+	if err != nil && err != io.EOF {
```

Comment:
IMHO, `extractYamlConfigurations` should not return `io.EOF` if it's meaninigless.